src/Microsoft.HttpRepl/Formatting/JsonVisitor.cs

```diff
@@ -11,104 +12,109 @@ namespace Microsoft.HttpRepl.Formatting
 {
     public static class JsonVisitor
     {
+#pragma warning disable CA1308 // Normalize strings to uppercase
```

Comment:
nit: I'm not a big fan of looong #pragma regions, it's easy to refactor and not realize the pragma was in effect.  Worth just suppressing with attribute on this method?

Reply:
Yeah, I should have done it that way to begin with. Done.